examples/cpu/cmake/AddISPCExampleModern.cmake

```diff
@@ -33,6 +33,7 @@ function(add_ispc_example)
         CXX_STANDARD_REQUIRED YES)
 
     set_property(TARGET ${example_NAME} PROPERTY POSITION_INDEPENDENT_CODE ON)
+    message(STATUS "ISPC_TARGETS: ${ISPC_TARGETS}")
```

Comment:
Is it here for some debug purposes?  If not, please add "The example ${example_NAME} is built for ISPC_TARGETS: ${ISPC_TARGETS}". Otherwise it looks weird in the console.

Reply:
Yes, it is for debug. I have removed it.